apps/condo/domains/meter/access/Meter.js

```diff
@@ -61,44 +62,96 @@ async function canManageMeters (args) {
         return await canManageObjectsAsB2BAppServiceUser(args)
     }
 
+    const isBulkRequest = Array.isArray(originalInput)
```

Comment:
i think it is possible to use this and get rid of else branch
```suggestion
    const inputs = Array.isArray(originalInput) ? originalInput : [originalInput]
```